packages/babel-plugin/src/__tests__/css-builder.test.ts

```diff
@@ -49,25 +49,4 @@ describe('css builder', () => {
       "
     `);
   });
-
```

Comment:
Moved to [behaviour.test.ts](https://github.com/atlassian-labs/compiled/pull/1139/files#diff-db85bb77fe1990a0e351b91b67d37bb7430d43fad07248e0da1192bb519eb115)